website/templates/index.html

```diff
@@ -34,14 +34,6 @@
     <!-- Font Awesome v.5.0.12 -->
     <link rel="stylesheet" href="/static/css/all.css" crossorigin="anonymous">
 
-    <!-- jquery datatable -->
```

Comment:
datatable wasn't working and shouldn't actually be necessary to have multiple pages if it's now fed by a search